coordinator/src/test/scala/filodb.coordinator/DatasetCoordinatorActorSpec.scala

```diff
@@ -0,0 +1,135 @@
+package filodb.coordinator
+
+import akka.actor.{ActorSystem, ActorRef, PoisonPill}
+import akka.testkit.TestProbe
+import akka.pattern.gracefulStop
+import com.typesafe.config.ConfigFactory
+import org.velvia.filo.{RowReader, TupleRowReader}
+import scala.concurrent.Future
+import scala.concurrent.duration._
+
+import filodb.core._
+import filodb.core.metadata.{Column, Dataset, RichProjection}
+import filodb.core.columnstore.{InMemoryColumnStore, SegmentSpec}
+import filodb.core.reprojector.{DefaultReprojector, MemTable, Reprojector}
+
+import org.scalatest.concurrent.ScalaFutures
+import org.scalatest.time.{Millis, Span, Seconds}
+
+object DatasetCoordinatorActorSpec extends ActorSpecConfig
+
+class DatasetCoordinatorActorSpec extends ActorTest(DatasetCoordinatorActorSpec.getNewSystem)
+with ScalaFutures {
+  import akka.testkit._
+  import SegmentSpec._
+  import DatasetCoordinatorActor._
+
+  implicit val defaultPatience =
+    PatienceConfig(timeout = Span(10, Seconds), interval = Span(50, Millis))
+
+  import scala.concurrent.ExecutionContext.Implicits.global
```

Comment:
Does ActorTest have an ActorSystem? I'm guessing yes because you call 'TestProbe()' which requires an implicit system. In which case I think you only need:

     import system.dispatcher